tensorflow/core/kernels/mkl/mkl_matmul_op_fused.cc

```diff
@@ -290,30 +308,31 @@ class MklFusedMatMulOp : public MklDnnMatMulOpBase<T, void, T> {
     }
   }
 
-  void ExtendMklDnnMatMulFwdParams(OpKernelContext* ctx,
-                                   MklDnnMatMulFwdParams& params) {
+  virtual void ExtendMklDnnMatMulFwdParams(OpKernelContext* ctx,
+                                           MklDnnMatMulFwdParams& params) {
+    // Create a string from data types of input, weight, bias, and output.
+    params.dtypes.append(typeid(T1).name());
+    params.dtypes.append(typeid(T2).name());
+    params.dtypes.append(typeid(Tbias).name());
+    params.dtypes.append(typeid(Toutput).name());
     if (fused_ops_.size() == 2) {
       string post_op = fused_ops_[1];
-
-      if (post_op == "Relu") {
-        params.post_op_params.push_back({"relu", {1.0, 0.0, 0.0}});
-      } else if (post_op == "Relu6") {
-        params.post_op_params.push_back({"relu6", {1.0, 6.0, 0.0}});
+      float scale = 1.0f;
+      float alpha = 0.0f;
+      float beta = 0.0f;
+      if (post_op == "Relu6") {
```

Comment:
Why has the capitalization changed?

Reply:
The purpose is to (1) directly match the string  from `fused_ops` attribute and (2) reduce `if else` conditions. 
Please see the removed codes: 303-309 and 312-316 and combine as single if in added code 328-333.